libs/tunables/tunables_shell.py

```diff
@@ -0,0 +1,201 @@
+"""The python shell interface that users interact with"""
+
+import cmd
+from ast import literal_eval
+from datetime import date
+import can_layer as CAN_layer
```

Comment:
Why do you have the import like this? Why not just do `from can_layer import TunablesCAN`?